src/__tests__/e2e/valid-mtls-enabled-and-oauth-enabled-3.2.0/serverless.yml

```diff
@@ -0,0 +1,91 @@
+service: serverless-wso2-apim
+provider:
+  name: aws
+  stackName: ${env:STACK_NAME}
+  deploymentBucket:
+    name: ${env:TEST_ID_NORMALIZED}
+plugins:
+  - serverless-localstack
+  - serverless-deployment-bucket
+  - "../../../../../src"
+
+#⬇⬇⬇⬇⬇⬇⬇⬇⬇⬇⬇⬇⬇⬇ Modify the configuration below to suit your test case.
```

Comment:
Can you remove unnecessary comments?

Reply:
that is in all e2e test cases, for the sake of keeping things same I did not touch it. Do you want me to remove it?